.packit.yml

```diff
@@ -13,14 +13,19 @@ jobs:
     trigger: pull_request
     metadata:
       targets:
-        - fedora-rawhide
+        - fedora-34
```

Comment:
Would `fedora-development` alias ([docs](https://packit.dev/docs/configuration/#available-copr-build-targets)) help to not need to change this every 6 months? (Will lead to `rawhide` or `rawhide + branched fedora`.)

Reply:
OK, I now see you have a dedicated branch for `f34` and my comment does not make much sense then.

Reply:
Hah, that's great improvement. Thanks for pointing that!